integration-tests/api/__tests__/admin/order.js

```diff
@@ -856,8 +926,59 @@ describe("/admin/orders", () => {
         expect.objectContaining({
           id: "test-order",
         }),
+        expect.objectContaining({
+          id: "test-order-w-c-and-s",
+        }),
       ]);
     });
+
+    it("Only allows canceling order after canceling contained elements", async () => {
```

Comment:
This follows the same strategy as with the Claim. This is however a bit larger, as the Order contains more different elements